client/components/suggestion-search/README.md

```diff
@@ -8,8 +8,8 @@ SuggestionSearch is a bundled component of FormTextInput and Suggestions which e
 ```es6
 import SuggestionSearch from 'components/suggestion-search';
 
-onChange( newValue ) {
-	console.log( 'New value: ', newValue );
+onChange( newValue, isNavigating ) {
```

Comment:
Interesting. I've solved this a different way in my branch by passing down both `onChange` and `onSelect` methods, the latter being a callback when a user selects a value rather than changing the field value. 

This is also works, but I'll leave my version in when it goes up for review and remind myself to ping you for review.

Reply:
Interesting. I'm actually not very satisfied with this `isNavigating` thing, because of the boolean blindness. Please feel free to keep your approach.